applications/FSIapplication/custom_utilities/variable_redistribution_utility.cpp

```diff
@@ -190,13 +208,18 @@ void VariableRedistributionUtility::SpecializedDistributePointValues(
         node_iter->FastGetSolutionStepValue(rDistributedVariable) = node_iter->FastGetSolutionStepValue(rPointVariable) / node_iter->GetValue(NODAL_MAUX);
     }
 
+    // Make sure that the initial approximation is the same between processes
+    rModelPart.GetCommunicator().SynchronizeVariable(rDistributedVariable);
+
     // Iteration: LumpedMass * delta_distributed = point_value - ConsistentMass * distributed_old
+    double error_l2_norm = 0.0;
     unsigned int iteration = 0;
     while ( iteration < MaximumIterations )
     {
         UpdateDistributionRHS<TPointNumber,TValueType>(rModelPart,rPointVariable, rDistributedVariable, mass_matrix);
 
-        double error_l2_norm = SolveDistributionIteration(rModelPart,rDistributedVariable);
+        error_l2_norm = SolveDistributionIteration(rModelPart,rDistributedVariable);
+        rModelPart.GetCommunicator().MaxAll(error_l2_norm);
```

Comment:
I'd do a sum, to be equivalent with the serial version.

Reply:
Done.